src/main/java/serverutils/integration/vp/VPClaimsLocation.java

```diff
@@ -72,8 +83,14 @@ public String teamHint() {
         }
     }
 
-    public EnumTeamColor getTeamColor() {
-        return color;
+    public String toggleLoadHint() {
+        return EnumChatFormatting.DARK_GRAY + "Double click to load/unload.";
```

Comment:
It would be nice to have these strings localizable.

Reply:
Good idea, thanks :)